src/MSON.h

```diff
@@ -135,7 +135,8 @@ namespace mson {
         FixedTypeAttribute    = (1 << 2),  // The type is fixed
         SampleTypeAttribute   = (1 << 3),  // The type is a sample
         DefaultTypeAttribute  = (1 << 4),  // The type is default
-        NullableTypeAttribute = (1 << 5)   // The type is nullable
+        NullableTypeAttribute = (1 << 5),  // The type is nullable
+        TypedTypeAttribute    = (1 << 6)   // The type is typed
```

Comment:
Let's change the comments a bit more since the type attributes are much more now.